common/src/lib.rs

```diff
@@ -20,19 +20,16 @@
 
 pub mod native;
 pub mod storage_queue;
+mod traits;
```

Comment:
As @NikVolf always says, it isn't the best naming practice to call modules like `structs`, `traits` etc.

Reply:
In the end of the day just reverted the creation of this separate module for traits and placed everything back into the `lib.rs` where it belongs.